### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by Androbin**
### Changes

- Fixed the `.json` file glob in `.gitignore`
- Removed unused/irrelevant `node_modules/` and `package-lock.json`
- Removed redundant/duplicate `__pycache__/` and `*.pyc`
- Removed redundant/duplicate `auto_gpt_workspace/`

### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [x] I have thoroughly tested my changes with multiple different prompts.
- [x] I have considered potential risks and mitigations for my changes.
- [x] I have documented my changes clearly and comprehensively.
- [x] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

**Review by blankster — DISMISSED** [review]

**Review by SuperYeti — DISMISSED** [review]
Looks good.

File: .gitignore [hunk 1]
```diff
@@ -1,8 +1,7 @@
 scripts/keys.py
-scripts/*json
-scripts/node_modules/
-scripts/__pycache__/keys.cpython-310.pyc
```
Comment (Torantulino) [inline .gitignore]: You also removed `keys.cpython-310.pyc`.

**Review by Torantulino — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Torantulino) [reply .gitignore]: Ah I see *.pyc is already ignored

Comment (Androbin) [reply .gitignore]: `scripts/__pycache__/keys.cpython-310.pyc` is ignored by virtue of both `__pycache__/` and `*.pyc`

**Review by nponeccop — APPROVED** [review]

**Review by nponeccop — DISMISSED** [review]

**Comment by nponeccop** [comment]
@Androbin There are conflicts now

**Comment by nponeccop** [comment]
We don't use npm or node do we?

**Comment by Androbin** [comment]
@nponeccop You're gonna have to ask @Torantulino about that (https://github.com/Significant-Gravitas/Auto-GPT/commit/217991eaf27119a749ccaf00fe5261c9432610f0)

**Comment by nponeccop** [comment]
@p-i- @richbeales Can you review?

File: .gitignore [hunk 2]
```diff
@@ -1,11 +1,6 @@
 ## Original ignores
 autogpt/keys.py
-autogpt/*json
-autogpt/node_modules/
-autogpt/__pycache__/keys.cpython-310.pyc
-autogpt/auto_gpt_workspace
```
Comment (ntindle) [inline .gitignore:6]: Put this one bac

**Review by ntindle — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 3]
```diff
@@ -1,11 +1,6 @@
...
-package-lock.json
-*.pyc
+autogpt/*.json
 auto_gpt_workspace/*
```
Comment (Androbin) [inline .gitignore:4]: @ntindle Is `autogpt/auto_gpt_workspace` not a duplicate of `auto_gpt_workspace/*` ?

**Comment by p-i-** [comment]
This is a mass message from the AutoGPT core team.
Our apologies for the ongoing delay in processing PRs.
This is because we are re-architecting the AutoGPT core!

For more details (and for infor on joining our Discord), please refer to:
  https://github.com/Significant-Gravitas/Auto-GPT/wiki/Architecting

**Comment by lc0rp** [comment]
Please merge [this PR](https://github.com/Androbin/Auto-GPT/pull/1) to bring this up to date.

**Comment by Androbin** [comment]
@lc0rp Done, thank you

**Review by ntindle — APPROVED** [review]

**Comment by libaxuan** [comment]
如何独接百度敏感词&nbsp;





------------------ Original ------------------
From: Nicholas Tindle ***@***.***&gt;
Date: Wed,Jun 7,2023 1:13 PM
To: Significant-Gravitas/Auto-GPT ***@***.***&gt;
Cc: Subscribed ***@***.***&gt;
Subject: Re: [Significant-Gravitas/Auto-GPT] Update .gitignore (PR #882)





 
Merged #882 into master.

**Pull request merged**